ansible/roles/synapse-extra-config/templates/cm-extraconfig.yml.j2

```diff
@@ -41,3 +41,16 @@ data:
     app_service_config_files:
       - /data/msteams-registration.yaml
 {% endif %}
+    media_storage_providers:
+      - module: s3_storage_provider.S3StorageProviderBackend
+        store_local: True
+        store_remote: True
+        store_synchronous: True
+        config:
+          bucket: {{ env }}-{{ matrix.s3_media_repo.bucket_name }}
+          # All of the below options are optional, for use with non-AWS S3-like
+          # services, or to specify access tokens here instead of some external method.
+          region_name: gra
```

Comment:
gra must not be hardcoded here.
The value must come from GLOBALE_REGION at the terraform level and be exported and transmitted to ansible through the generate_configuration_var_files.sh script